Caddyfile.base

```diff
@@ -0,0 +1,4 @@
+localhost {
```

Comment:
oups wrong name, should be `pr-list.joinplu.me` or something like that (i forgot to set it back after testing)